### PR TITLE
Fixed filtering enqueue by post type

### PR DESCRIPTION
**Pull request by giwrgos88**
Method was always returning true so filtering wasn't working

### CODE REVIEW

File: Herbert/Framework/Enqueue.php [hunk 1]
```diff
@@ -327,7 +327,7 @@ public function filterSearch($attrs, $filterWith)
      */
     public function filterPostType($attrs, $filterWith)
     {
-        return array_search(get_post_type(), $filterWith) !== null;
+        return array_search(get_post_type(), $filterWith) !== FALSE ? TRUE : FALSE;
```
Comment (jeremyzahner) [inline Herbert/Framework/Enqueue.php]: The ternary operator is useless for booleans.

``` php
return array_search(get_post_type(), $filterWith) !== false;
```

Comment (jeremyzahner) [reply Herbert/Framework/Enqueue.php]: So please remove `? TRUE : FALSE` since there is no point in having that.

**Review by giwrgos88 — COMMENTED** [review]
Removed ternary operator

**Comment by jeremyzahner** [comment]
@giwrgos88 Thanks!

**Pull request merged**